### DISCUSSION TITLE
--jvquoteall not necessarily useless?

### DISCUSSION THREAD

**Discussion by trantor** [johnkerl/miller#1079] [Q&A] [answered]
Hi @johnkerl !
Porting my code to miller 6 I noticed how the option in question has been turned into a no-op.
I understand the reasoning in the docs, however in my case I used it to force everything as a string in the output, even integer values.
Which means that the modification turned into a compatibility-breaking situation.
Did you consider this scenario?

**Comment by johnkerl** [comment]
@trantor in Miller 6 I'm using a JSON library which I have less control over :( 

I just thought though ... maybe if `--jvquoteall` I can coerce all values to strings before handling off to the golang JSON encoder -- ? Seems like it would work ...

**Reply by aborruso** [reply]
In recent weeks, several co-workers have found it uncomfortable not to have `--quote-all` anymore, for CSV export.

**Reply by johnkerl** [reply]
@aborruso that meshes well with https://github.com/johnkerl/miller/issues/1026 -- thank you! :)

**Reply by johnkerl** [reply]
@trantor #1083

**Reply by johnkerl** [reply]
@aborruso #1084

**Comment by johnkerl** [comment, marked as answer]
#1083 restores this
